### PR TITLE
Frog Jump solution

### PR DESCRIPTION
**Pull request by flackovic**
Solution to the Frog Jump problem.

Calculates minimum jumps needed to go from point A to point B given the jump length.

### CODE REVIEW

File: tests/FrogJmpTest.php [hunk 1]
```diff
@@ -32,10 +32,12 @@ public function testCalculateWorks($x, $y, $d, $expected)
     public function provideTestCalculateWorks()
     {
         return [
-            [10, 85, 30, 3],
+            [10, 10, 30, 0],
```
Comment (HcsOmot) [inline tests/FrogJmpTest.php]: nice catch! 👍

**Comment by HcsOmot** [comment]
You should clean up the assignment commit history - I've noticed you pulled back in the percentage calculator code.
Let me know if you need help! 👍

File: src/FrogJmp.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * Date: 30.11.17.
+ * Time: 10:05
+ */
+
+class FrogJmp {
+
+    public function solution($x, $y, $d)
+    {
+        $jumps = floor(($y - $x) / $d);
+        return ($y - $x) % $d == 0 ? $jumps : $jumps + 1;
```
Comment (franraknic) [inline src/FrogJmp.php]: ako se zaokruzi na gore sa  `ceil()` onda ovo dolje nije potrebno nego možemo jednostavno castati sa (int)
također, return sa ternarnim možda i nije najčitljivije

**Comment by flackovic** [comment]
Riješeno.

**Pull request closed without merging**